cypress/fixtures/devices.json

```diff
@@ -0,0 +1,67 @@
+{
+  "data": [
```

Comment:
we could add one or two more devices, even if with fewer data. Take for example a device that just got registered, but never connected to astarte